ui/tests/helpers/kv/kv-selectors.js

```diff
@@ -24,14 +26,14 @@ export const PAGE = {
     secretMetadataSection: '[data-test-kv-metadata-section]',
   },
   detail: {
-    versionTooltip: '[data-test-kv-version-tooltip-trigger]',
+    versionCreated: '[data-test-kv-version-tooltip-trigger]',
     versionDropdown: '[data-test-version-dropdown]',
     version: (number) => `[data-test-version="${number}"]`,
     createNewVersion: '[data-test-create-new-version]',
   },
   list: {
     createSecret: '[data-test-toolbar-create-secret]',
-    item: (secret) => `[data-test-list-item="${secret}"]`,
+    item: (secret) => (!secret ? '[data-test-list-item]' : `[data-test-list-item="${secret}"]`),
```

Comment:
If no secret passed in, it will find all of them which allows us to count how many items